ddl/callback.go

```diff
@@ -142,6 +143,119 @@ func newDefaultCallBack(do DomainReloader) Callback {
 
 // ****************************** End of Default DDL Callback Instance *********************************************
 
+// ****************************** Start of Test DDL Callback Instance ***********************************************
```

Comment:
This is just a code move in the file, for compatibility with the current PR test call issue.